contracts/integrations/test/framework-unit-tests/deployment_manager_test.ts

```diff
@@ -79,14 +70,24 @@ blockchainTests('Deployment Manager', env => {
 
             // The other asset proxies should have the exchange and the multi-asset proxy in their
             // authorities list.
-            await batchAssertAuthoritiesAsync(
-                [deploymentManager.assetProxies.multiAssetProxy.address, deploymentManager.exchange.address],
-                [
-                    deploymentManager.assetProxies.erc1155Proxy,
-                    deploymentManager.assetProxies.erc20Proxy,
-                    deploymentManager.assetProxies.erc721Proxy,
-                ],
-            );
+            const erc20ProxyAuthorities = await deploymentManager.assetProxies.erc20Proxy.getAuthorizedAddresses.callAsync();
```

Comment:
What was the reasoning behind this change?

Reply:
`zrxVault` is now an authority for `erc20Proxy`

Reply:
Sure, but why did you remove the helper function? It could just be added to the array?

Reply:
the helper function assumes a complete bipartite graph between authorities and asset erc*proxy contracts. zrxVault is only authorized for erc20Proxy

Reply:
Oh I see. 

Carry on!